container-search/src/main/resources/configdefinitions/llm-local-client.def

```diff
@@ -0,0 +1,32 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package=ai.vespa.llm.clients
+
+# Url to the model to use
```

Comment:
Use a "model" parameter type instead of url to have the path, url and cloud options handled as one.